### PR TITLE
EZP-27191: Add ezsystems/ezplatform-multi-file-upload package into the ezplatform meta repository

### PR DESCRIPTION
**Pull request by lserwatka**
https://jira.ez.no/browse/EZP-27191

### CODE REVIEW

File: app/config/routing.yml [hunk 1]
```diff
@@ -30,3 +30,7 @@ _ezplatformRepositoryFormsRoutes:
 _contentOnTheFly:
     resource: "@ContentOnTheFlyBundle/Resources/config/routing.yml"
     prefix:   "%ezpublish_rest.path_prefix%"
+
+_multiFileUpload:
+    resource: "@EzSystemsMultiFileUploadBundle/Resources/config/routing.yml"
+    prefix:   "%ezpublish_rest.path_prefix%"
```
Comment (andrerom) [inline app/config/routing.yml]: nitpick: new line

File: composer.json [hunk 2]
```diff
@@ -40,7 +40,8 @@
         "doctrine/doctrine-bundle": "~1.5",
         "sensio/framework-extra-bundle": "~3.0",
         "ezsystems/content-on-the-fly-prototype": "~0.1.7",
-        "willdurand/js-translation-bundle": "^2.6.4"
+        "willdurand/js-translation-bundle": "^2.6.4",
+        "ezsystems/ezplatform-multi-file-upload": "^0.1.0@dev"
```
Comment (bdunogier) [inline composer.json]: Could you move that requirement a bit higher ? The goal is to reduce the likeliness of conflicts for our users. The last one that was added is `ezplatform-i18n/ezplatform-i18n-ach_ug`, this one should go right below it.

File: app/config/routing.yml (hunk 1 above)
Comment (lserwatka) [inline app/config/routing.yml]: I added new line but for some reason Github cannot see it.

**Comment by bdunogier** [comment]
One retrospect question: with these changes alone, the package won't be usable out-of-the-box, will it ? It requires that the [configuration](https://github.com/ezsystems/ezplatform-multi-file-upload#configuration) is added to the config, does it not ?

Should we consider some default configuration ?

**Pull request merged**